src/test/ui/consts/inline_asm.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0019]: constant contains unimplemented expression type
+  --> $DIR/inline_asm.rs:3:1
+   |
+LL | const _: () = unsafe { llvm_asm!("nop") };
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I wonder why all these spans are so bad. Ah, looks like  `visit_statement` does not call `super_statement` except for `Assign`.

Reply:
Please walk it for `LlvmInlineAsm`, and if you feel motivated, try out what happens when you do the same for the `IfOrMatch` arm